calendars/skimage.yaml

```diff
@@ -6,12 +6,12 @@ events:
 
       Meeting notes: https://hackmd.io/@alexdesiqueira/skimage_cc
       Archive — Meeting Notes: https://github.com/scikit-image/meeting-notes
-    begin: 2022-06-14 17:00:00 +00:00
-    end: 2022-06-14 18:00:00 +00:00
-    url: https://meet.google.com/icq-geqg-yix
+    begin: 2022-08-23 17:00:00 +00:00
+    end: 2022-08-23 18:00:00 +00:00
```

Comment:
```suggestion
    begin: 2022-08-30 17:00:00 +00:00
    end: 2022-08-30 18:00:00 +00:00
```

Reply:
Thanks for reviewing @jarrodmillman! The idea was to have a meeting every other week.
Let me try to fix that.